products/surface.md

```diff
@@ -183,7 +238,7 @@ releases:
 
 -   releaseCycle: Surface RT
     releaseDate: 2012-10-26
-    eol: 2017-04-11
+    eol: 2023-01-10
```

Comment:
this one is also EOLed : https://support.microsoft.com/en-us/surface/surface-rt-update-history-9339a091-a253-0511-f61a-84313af4487d

Reply:
Fixed.